addons/web/static/src/model/relational_model/utils.js

```diff
@@ -393,6 +393,13 @@ export function getFieldsSpec(activeFields, fields, evalContext, { orderBys, wit
             case "reference": {
                 fieldsSpec[fieldName].fields = {};
                 if (!isAlwaysInvisible) {
```

Comment:
for x2many, I see that we have a flag `withInvisible`. I don't know what it is (or don't remember), but I assume it should be use here as well